docs/docs/data-structures/tree/node-types.mdx

```diff
@@ -7,27 +7,41 @@ Data on a [SharedTree](./index.mdx) is stored as a [node](./nodes.mdx) of one of
+A [leaf](https://fluidframework.com/docs/api/fluid-framework/nodekind-enum#leaf-enummember) node represents an atomic value, while a [TreeNodes](https://fluidframework.com/docs/api/fluid-framework/treenode-class) represents an object or collection.
 The following leaf node types are available:
 
--   **boolean**: Works identically to a JavaScript boolean.
--   **number**: Works identically to a JavaScript JavaScript number.
--   **string**: Works identically to a JavaScript string.
--   **null**: Works identically to a JavaScript null.
--   **FluidHandle**: A handle to a Fluid DDS or Data Object in the current container. For more information about handles see [Handles](../../concepts/handles.mdx).
+-   [**boolean**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#boolean-property): Works identically to a JavaScript boolean.
+-   [**number**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#number-property): Works identically to a JavaScript JavaScript number except does not support -0, NaN or Infinity.
+-   [**string**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#string-property): Works identically to a JavaScript string except for not supporting unpaired UTF-16 surrogate pairs.
+-   [**null**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#null-property): Works identically to a JavaScript null.
+-   [**FluidHandle**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#handle-property): A handle to a Fluid DDS or Data Object in the current container. For more information about handles see [Handles](../../concepts/handles.mdx).
 
-The following subsections describe the available internal node types.
+## TreeNodes
 
-## Object Nodes
+The following subsections describe the available [TreeNode](https://fluidframework.com/docs/api/fluid-framework/treenode-class) types.
 
-An object node is a TypeScript-like object with one or more named child properties. The object node's properties can in principle be any of the node types including internal node types; but typically the schema for the `SharedTree` that your code defines will specify for any object node a specific set of properties and node types of each. A `SharedTree` can have many object nodes at various places in the tree and they do not all have to conform to the same schema. Your schema can specify different properties for different object nodes. The schema also specifies whether a child property is required or optional, and it can assign a union datatype to any property. For example, a property could be either number or string.
+### Object Nodes
 
-## Map Nodes
+An [object node](https://fluidframework.com/docs/api/fluid-framework/treeobjectnode-typealias) is a JavaScript object like TreeNode with zero or more named child [fields](https://fluidframework.com/docs/api/fluid-framework/fieldschema-class) exposed as properties of the object.
```

Comment:
Why does the document say `TreeObjectNode TypeAlias` and not just `TreeObjectNode`?